cord19-explorer/src/main/java/edu/asu/diging/cord19/explorer/core/model/impl/AuthorImpl.java

```diff
@@ -0,0 +1,31 @@
+package edu.asu.diging.cord19.explorer.core.model.impl;
+
+public class AuthorImpl {
+
+    private String lastName;
+    private String firstName;
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+    
+    @Override
+    public boolean equals(Object obj) {
+        AuthorImpl name = (AuthorImpl) obj;
+        return (this.firstName+this.lastName).equals(name.getFirstName()+name.getLastName());
+        
+    }
```

Comment:
you need to override equals and hashcode. Just let Eclipse generate those for you (no coding needed).